### PR TITLE
[Express Route] Added Support for IPv6 Circuit Connection Configuration.

### PR DESCRIPTION
**Pull request by amwate**
### Latest improvements:
Below changes to ExpressRouteCircuitConfig are to provide the support for IPv6 properties. 
The support was added to NRP in the September release. Although the feature was not published or GA'd.

The below configurations would add a node IPv6CircuitConnectionProperties to the existing expressRouteCircuitConnection object. 

The above block would be used by all the northbound clients like azure-powershell/ azcli/terraform/portal.

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/expressRouteCircuit.json [hunk 1]
```diff
@@ -1948,6 +1948,18 @@
         "modelAsString": true
       }
     },
+    "Ipv6CircuitConnectionConfig": {
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/expressRouteCircuit.json]: Please add `description`

File: specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/expressRouteCircuit.json [hunk 2]
```diff
@@ -1948,6 +1948,18 @@
         "modelAsString": true
       }
     },
+    "Ipv6CircuitConnectionConfig": {
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/expressRouteCircuit.json]: Please add `description`

**Review by anton-evseev — CHANGES_REQUESTED** [review]
Please address `ModelValidation` errors
https://dev.azure.com/azure-sdk/public/_build/results?buildId=233338&view=logs&jobId=8d2f9c49-cd83-5a9d-f3fe-2fd651afa742

File: specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/expressRouteCircuit.json (hunk 1 above)
Comment (amwate) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/expressRouteCircuit.json]: Added

File: specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/expressRouteCircuit.json (hunk 2 above)
Comment (amwate) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/expressRouteCircuit.json]: Added

**Comment by MikhailTryakhov** [comment]
Hey @amwate, we are close to release our monthly branch. In case you won't solve all the failed jobs/get ARM approval - your PR would be closed around midnight today.

**Review by v-shhatt — COMMENTED** [review]
reviewed.. please fix errors

**Comment by number213** [comment]
`network-december-release` has been already merged to meep tight deadlines. If your PR is urgent, please retarget to `master`

**Comment by amwate** [comment]
> `network-december-release` has been already merged to meep tight deadlines. If your PR is urgent, please retarget to `master`

It is not urgent. We can retarget it to network-january-release when it is pulled.

**Comment by amwate** [comment]
> Hey @amwate, we are close to release our monthly branch. In case you won't solve all the failed jobs/get ARM approval - your PR would be closed around midnight today.

Sure no problem. We can targe this in the next release.

**Comment by MikhailTryakhov** [comment]
Closing per latest comment

**Pull request closed without merging**